apps/web/src/app/globals.css

```
@@ -1,6 +1,6 @@
 @import "tailwindcss";
 
-@theme inline {
+@theme {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)